docs/release-notes/v11.1.0.md

```diff
@@ -0,0 +1,136 @@
+
+_May 25th, 2023_
+
+## Console
+
+### New Features
+
+#### User Groups Management
+
+From a dedicated section inside of the Company Overview, Company Owners can now manage Company users also through Groups.  
+Thanks to Groups it is possible to simplify the management of a multitude of users who have to be assigned the same privileges (e.g. a work team).  
+Management of users through groups sits alongside the actual individual management of users, to allow you to better structure access to resources as you prefer, giving you the freedom to choose how to govern access levels of your Company users.  
+
+To discover more about user groups management, just take a look at the related [documentation page](../development_suite/identity-and-access-management/group-management.md). 
+
+# Privatized Marketplace for Companies
+
+From now on, Companies can have their own private Marketplace items, which will not be visible and usable from those who are not part of those specific Companies.   
+More specifically, Company Owners and Project Administrators can now be able to manage – directly from CMS – Marketplace resources that will be available exclusively for the members of their Company. 
+This has to be done by specifying the `tenantId` as a Company identifier. 
+
+To discover more information, visit the dedicated Marketplace [documentation page](../marketplace/add_to_marketplace/contributing_overview#public-and-private-marketplace.md).
+
+### Improvements
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* An error that caused the Debug Area of the Console to fail to load has been fixed
+* A bug in the Project creation that caused a malfunction in the display of the default CI/CD tool of each environment has been fixed
+* An issue that prevented the user to understand the type of error running in a Pod, by showing a generic "undefined" in the information popup has been fixed
+* The Backoffice Configurator now correctly configures the `oneOf` properties, handling it properly in editing modal
+
+## Fast Data
+
+### New Features
+
+#### Ingestion Reloader
+
+Introducing a new version of the Ingestion Reloader, the `1.2.0`! With this new verison, you can now filter specific messages retrieved from a bucket, using a custom selector system and allowing precise and targeted filtering of messages to be reingested: just specify the file with it in the `customSelectorName` field. The selector's configurations are available at the POST `/reingestion/file` and POST `/reingestion/topic` endpoints.
+
+Go to the [official documentation](/fast_data/bucket_storage_support/configuration/ingestion_reloader.md) to find out more.
+
+#### Debezium DB2 plugin
+The rollout of the supported Debezium Plugins is going on! You can now find in the Marketplace the [Debezium connector plugin for DB2](/fast_data/connectors/debezium_cdc#db2.md) databases with the pre-configured config map for Debezium Server to connect directly to a DB2 Database.
```

Comment:
@giuliowaitforitdavide is it a template, not a plugin, am I right?

Reply:
Fixed